_posts/events/2024-08-09-pyopensci-fall-festival.md

```diff
@@ -19,4 +19,99 @@ type: "event"
 comments: true
 ---
 
-## More coming soon
+## <i class="fa-regular fa-heart"></i> Announcing the pyOpenSci Fall Festival!
+
+_This post last updated on 6 August, 2024_
+
+Happening October 28--November 01, 2024, the pyOpenSci Fall Festival is an online community training and networking event designed to empower scientists with in demand open science skills. Our goals with the Fall Festival are to empower the community with technically-relevant open science skills, build community, and help raise funds for pyOpenSci operations that allow us to continue supporting community work. Interested? Read on to learn more!
```

Comment:
```suggestion
Happening October 28--November 01, 2024, the pyOpenSci Fall Festival is an online community training and networking event designed to empower scientists with in demand open science skills. Our goals for the Fall Festival are to 

* empower you with technically-relevant open science skills, 
* call attention to and celebrate new and upcoming tools that support open reproducible science
* build community, and 
* help raise funds for pyOpenSci operations that allow us to continue supporting community work. 

Interested? Read on to learn more!
```

Reply:
i switched it to "you" - only because it had the word "community" so many times i wasn't sure who we were speaking too. The bigger question around fundraising as a goal. i love the transparency. do other organizations write that in their event pages? like psf and pycon? maybe they do! i just don't know what is best.